jupyterlab_server/settings_handler.py

```diff
@@ -16,6 +16,9 @@
 
 # The JupyterLab settings file extension.
 SETTINGS_EXTENSION = '.jupyterlab-settings'
+# Used for when JSON5 payload is wrapped in a JSON payload in the form:
```

Comment:
Since this is only used one place, I think we should just hardcode it there. Otherwise we have to cross-reference this one variable definition to understand the code below.

Reply:
Sure, that's understandable. I'll hardcode it.